supporting-blog-content/alternative-approach-for-parsing-pdfs-in-rag/README.md

```diff
@@ -0,0 +1,19 @@
+
+# PDF Parsing - Table Extraction
+
+ Python notebook demonstrates an alternative approach to parsing PDFs, particularly focusing on extracting and converting tables into a format suitable for search applications such as Retrieval-Augmented Generation (RAG). The notebook leverages Azure OpenAI to process and convert table data from PDFs into plain text for better searchability and indexing.
+
+## Features
+- **PDF Table Extraction**: The notebook identifies and parses tables from PDFs.
+- **LLM Integration**: Calls Azure OpenAI models to provide a text representation of the extracted tables.
+- **Search Optimization**: The parsed table data is processed into a format that can be more easily indexed and searched in Elasticsearch or other vector-based search systems.
+  
+## Getting Started
+
+### Prerequisites
+- Python 3.x
+
```

Comment:
Might be worth it to let the reader know here they need an Azure OpenAI  account with a deployment model endpoint and key. A link to where to get those would be helpful as well.

Reply:
done.